stdlib/Logging/src/Logging.jl

```diff
@@ -29,6 +29,41 @@ for sym in [
     @eval const $sym = Base.CoreLogging.$sym
 end
 
+"""
+    @create_log_macro(name::Symbol, level::Int, color::Union{Int,Symbol})
+
+Creates a custom log macro like `@info`, `@warn` etc. with a given `name`, `level` and
+`color`. The macro created is named with the lowercase form of `name` but the given form
+is used for the printing.
+
+See `Base.text_colors` for recognized color values.
```

Comment:
```suggestion
See [`Base.text_colors`](@ref) for recognized color values.
```

Reply:
This isn't specifically documented, but it is mentioned elsewhere in the docs like

> The available color keys can be seen by typing Base.text_colors in the help mode of the REPL